### PR TITLE
fix: change autogen behaviour #NTRN-397

### PR DESCRIPTION
**Pull request by joldie777**
Task: https://hadronlabs.atlassian.net/browse/NTRN-397

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,23 @@ git submodule update --init --recursive
...
+npm run codegen
+
+# Build .js/.d.ts files
+npm run build
+```
+
+### Removing all unnecessary folders and files
+
+```sh
+npm run clean
```
Comment (pr0n00gler) [inline README.md:41]: Why it is not done automatically after a build?

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -20,11 +20,23 @@ git submodule update --init --recursive
 
 # Install dependencies
 npm install
+
+# Set specified versions for submodules
+npm run set-versions
```
Comment (pr0n00gler) [inline README.md:25]: Please write in readme how where we should specify versions of different reposiories

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (joldie777) [reply README.md:26]: Done

File: scripts/set-versions.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+readonly NEUTRON_DIR="neutron-src"
+readonly SLINKY_DIR="slinky-src"
+readonly WASMD_DIR="wasmd-src"
+
+readonly ADMIN_MODULE_REV="v2.0.2"
+readonly BLOCK_SDK_REV="v2.1.5"
+readonly COSMOS_SDK_REV="v0.50.9-neutron"
+readonly FEEMARKET_REV="v1.1.1"
+readonly IBC_GO_REV="v8.5.1"
+readonly NEUTRON_REV="feat/choose-cron-blocker-2"
```
Comment (pr0n00gler) [inline scripts/set-versions.sh:20]: v4.2.4

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

File: scripts/set-versions.sh (hunk 3 above)
Comment (joldie777) [reply scripts/set-versions.sh]: Done

File: scripts/set-versions.sh [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+readonly NEUTRON_DIR="neutron-src"
+readonly SLINKY_DIR="slinky-src"
+readonly WASMD_DIR="wasmd-src"
+
+readonly ADMIN_MODULE_REV="v2.0.2"
+readonly BLOCK_SDK_REV="v2.1.5"
+readonly COSMOS_SDK_REV="v0.50.9-neutron"
+readonly FEEMARKET_REV="v1.1.1"
+readonly IBC_GO_REV="v8.5.1"
+readonly NEUTRON_REV="v4.2.4"
```
Comment (pr0n00gler) [inline scripts/set-versions.sh:20]: My bad, should be main. Regenerate all files as well

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

File: scripts/set-versions.sh (hunk 4 above)
Comment (joldie777) [reply scripts/set-versions.sh]: Done

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**